genie-core/src/test/java/com/netflix/genie/core/properties/ExponentialBackOffTriggerPropertiesUnitTest.java

```diff
@@ -0,0 +1,61 @@
+package com.netflix.genie.core.properties;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Unit test for ExponentialBackOffTriggerProperties.
+ *
+ * @author mprimi
+ * @since 3.3.9
+ */
+public class ExponentialBackOffTriggerPropertiesUnitTest {
```

Comment:
Any reason you did this with raw `junit` instead of `spock`?

Reply:
Just cloned an existing test for a sister class, figured consistency with existing would be better than switching to spock in this case.